doc/application/application.rst

```diff
@@ -674,9 +674,8 @@ Create a Debugger Configuration
 RTOS Awareness
 ==============
 
-Experimental support for Zephyr RTOS awareness is implemented in `pyOCD PR
-#333`_. It is compatible with GDB PyOCD Debugging in Eclipse, but you must
-download this pull request and build pyOCD from source. You must also enable
+Support for Zephyr RTOS awareness is implemented in `pyOCD v0.11.0`_ and later.
```

Comment:
PR title says v0.11.1 so should this say 0.11.1 as well?

Reply:
This feature was added in v0.11.0, but I tested with v0.11.1.